devicetypes/smartthings/zwave-water-sensor.src/zwave-water-sensor.groovy

```diff
@@ -58,12 +58,12 @@ metadata {
 }
 
 def initialize() {
-	if (zwaveInfo.mfr.equals("0086"))
-		// 8 hour (+ 2 minutes) ping for Aeotec
+	if (zwaveInfo.mfr.equals("0086") || zwaveInfo.mfr.equals("021F") || zwaveInfo.mfr.equals("0258"))
+	// 8 hour (+ 2 minutes) ping for Aeotec, NEO Coolcam and Dome
 		sendEvent(name: "checkInterval", value: (8 * 60 * 60) + 120, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
 	else
-		// 12 hours for other devices
-		sendEvent(name: "checkInterval", value: (2 * 12 + 2) * 60 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+	// 12 hours (+ 2 minutes) for other devices
+		sendEvent(name: "checkInterval", value: (12 * 60 * 60) + 120, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
```

Comment:
can you copy the formatting we're using for other time values here and make this `+ 2 * 60` and `60 * 60` below?

Reply:
Of course, done